sdk/keyvault/azure-security-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/JREKeystoreTest.java

```diff
@@ -0,0 +1,280 @@
+        "aW4wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQC/QWKhTKDVdZW0rdC8" +
+        "OLD/3oCrkS4UMh3jh81g8vjOetuO8pki94Aj+vfvhIdSqB8ddhh8LdilAWp9e03T" +
+        "LpgXf9DlVTVrkt7Yr7t1+V3YvSQonIf1RODSpLumHZY7VvIB2Hu9rDAxPGd4AXTx" +
+        "JSokOb4JBEXCTSves8a3vY6NxNwprlBzRpqC7q/MyvpKki92I7CvWB8SQLla9y7e" +
+        "zewUpf9Wkc2SixuDZ+xQNPZZv7DBynF1DB9wGkhfbcsUmQ5GD5ZD3fDiXwL6CoTF" +
+        "XRr+KAEZBMwmtBBj93q9se2dWE1KWFD9fRNjOHrl6+rAqaQKMcjwNt0K5jFpoSBl" +
+        "lJ35AgMBAAGjUzBRMB0GA1UdDgQWBBRJUbpVuE6UECeZZJswRaXb6rPHVDAfBgNV" +
+        "HSMEGDAWgBRJUbpVuE6UECeZZJswRaXb6rPHVDAPBgNVHRMBAf8EBTADAQH/MA0G" +
+        "CSqGSIb3DQEBCwUAA4IBAQBh4V0n0+Dvvsy05Gp6MWizqEVzIQNdVto/3OHpg1Bt" +
+        "pYsmj+GqZpO/ElcXJmGVvWnx8uN1bT+sugxlBzJME/j9inhv1LJ89vd0vXXDn62t" +
+        "jQNjLiKMj7VusK3xOfHcvQ78N+horyuVIU6NhZ5uT/xMz5AcovzwUA8vaS4jCRfK" +
+        "EHoGeQvjuAvPaUIWdVlDeD5t+rkupkysEeMIrMoVB969gt34Ki0NuxDxJE7YxEJq" +
+        "6rCfWN6NukKFTKgVhwOg5r0iNBf1ewmujaHWA9eX8Ug62yS1liQyQXQmJKlJWNqh" +
+        "aoEAAH9ED5yDWyHROIa1DEaEfI6MjXS+ZusfLctC1tf/";
+
+    @Test
+    public void testLocalKeystore() throws Exception {
+        //   Security.insertProviderAt(new KeyVaultJcaProvider(), 1);
+
+        Security.insertProviderAt(new KeyVaultTrustManagerFactoryProvider(), 1);
+        KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+        try (final FileInputStream inStream = new FileInputStream(getKeyStoreFile())) {
+            keyStore.load(inStream, keyStorePassword.toCharArray());
+        }
+        keyStore.setEntry(aliasStrategy.chooseAlias(null, null), new KeyStore.PrivateKeyEntry(getKey(), getCertChain()), new KeyStore.PasswordProtection(keyPassword.toCharArray()));
+        /*
+         * Setup server side.
+         */
+        int randomPort = ThreadLocalRandom.current().nextInt(49452, 52570);
+        logger.log(Level.INFO, "The port used is " + randomPort);
+        SSLContext sslContext = SSLContexts.custom()
+            .loadKeyMaterial(keyStore, keyPassword.toCharArray(), aliasStrategy)
+            .build();
+        SSLServerSocketFactory factory = sslContext.getServerSocketFactory();
+        SSLServerSocket serverSocket = (SSLServerSocket) factory.createServerSocket(randomPort);
+
+        Thread serverThread = new Thread(() -> {
+            while (true) {
+                try {
+                    Socket socket = serverSocket.accept();
+                    try (OutputStream outputStream = socket.getOutputStream()) {
+                        outputStream.write("HTTP/1.1 204\r\n".getBytes());
+                        outputStream.flush();
+                    }
+                } catch (IOException ioe) {
+                    logger.log(Level.SEVERE, "Server can't write to the client socket.", ioe);
+                }
+            }
+        });
+        serverThread.start();
+
+        /*
+         * Setup client side
+         */
+
```

Comment:
Please re-arrange the empty lines:
1. Delete unnecessary empty line.
2. Do not put 2 empty line together in normal case.